env/src/env.yml

```diff
@@ -0,0 +1,24 @@
+name: echo
+version: "0.0.0"
+author: Federico Ponzi <federico.ponzi92@gmail.com>
+about: "
+     env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]\n\n
+    Set each NAME to VALUE in the environment and run COMMAND.:\n\n"
+args:
+    - OPTIONS:
+        help: Environement variables in the form of NAME=VALUE and the COMMAND to run with its arguments.
+        multiple: true
+    - IGNORE_ENVIRONMENT:
```

Comment:
For flags and options, use the same name as long, I don't really mind the way you did, but all other utilities does that and consistency is good :3